extensions/binding/org.eclipse.smarthome.binding.triggertest/ESH-INF/thing/thing-types.xml

```diff
@@ -51,6 +51,7 @@
         <kind>trigger</kind>
         <label>Trigger 2</label>
         <description>Description of trigger 2</description>
+        <event/>
```

Comment:
Well, this would be a different semantic than for "state", which is imho not good. See https://www.eclipse.org/smarthome/documentation/development/bindings/xml-reference.html#xml-structure-for-thing-descriptions:
"state: The restrictions of an item state which gives information how to interpret it (optional)."